internal/pkg/build/buildkit/client/client.go

```diff
@@ -182,8 +200,16 @@ func startBuildkitd(ctx context.Context, opts *Opts) (bkSocket string, cleanup f
 			sylog.Errorf("while canceling buildkit daemon process: %v", err)
 		}
 		cmd.Wait()
+		if tmpRoot != "" {
+			sylog.Warningf("removing singularity-buildkitd temporary directory %s", tmpRoot)
+			err := os.RemoveAll(tmpRoot)
+			if err != nil {
+				sylog.Errorf("while removing singularity-buildkitd temp dir: %v", err)
+			}
```

Comment:
Nit: seems the rest of the similar code had this format.

```suggestion
			if err := os.RemoveAll(tmpRoot); err != nil {
				sylog.Errorf("while removing singularity-buildkitd temp dir: %v", err)
			}
```